core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/deploysemantics/PhysicalOp.scala

```diff
@@ -457,12 +456,10 @@ case class PhysicalOp(
             )
           )
         }
-        val ref =
-          controllerActorService.actorOf(
-            workflowWorker.withDeploy(Deploy(scope = RemoteScope(preferredAddress)))
-          )
-        actorRefService.registerActorRef(workerId, ref)
-        opExecution.getWorkerInfo(workerId).ref = ref
+        controllerActorService.actorOf(
```

Comment:
add a comment to explain that we ignore the ref returned here?

Reply:
added